forecast_model.R

```diff
@@ -87,10 +87,11 @@ forecast <- NULL
       fit <- lm(site_target$temperature~site_target$air_temperature)
 
       #use linear regression to forecast water temperature for each ensemble member
-      forecasted_temperature <- fit$coefficients[1] + fit$coefficients[2] * noaa_future_site$air_temperature
+      forecasted_temperature <- fit$coefficients[1] + 
+        fit$coefficients[2] * noaa_future_site$air_temperature
 
       #use forecasted temperature to predict oyxgen by assuming that oxygen is saturated.
-      forecasted_oxygen <- rMR::Eq.Ox.conc(forecasted_temperature, elevation.m = ,site_info$field_mean_elevation_m,
+      forecasted_oxygen <- rMR::Eq.Ox.conc(forecasted_temperature, elevation.m = site_info$field_mean_elevation_m,
```

Comment:
little typo, not sure what the consequence was, weird that this didn't throw an error.